### PR TITLE
Generate File{Input,Output,IO}Stream automatically

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by sdroege** [comment]
CC @MathieuDuponchelle @EPashkin

File: Gir.toml [hunk 1]
```diff
@@ -64,15 +64,15 @@ generate = [
     "Gio.ThemedIcon",
     "Gio.TlsCertificate",
     "Gio.TlsCertificateFlags",
+    "Gio.FileIOStream",
+    "Gio.FileInputStream",
+    "Gio.FileOutputStream",
```
Comment (EPashkin) [inline Gir.toml]: Please insert it in such a way that the sorting is preserved

File: src/auto/file_i_o_stream.rs [hunk 2]
```diff
@@ -1,15 +1,15 @@
-// Copyright 2013-2017, The Gtk-rs Project Developers.
-// See the COPYRIGHT file at the top-level directory of this distribution.
-// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+// This file was generated by gir (https://github.com/gtk-rs/gir @ bd67955)
+// from gir-files (https://github.com/gtk-rs/gir-files @ db49619)
+// DO NOT EDIT
```
Comment (EPashkin) [inline src/auto/file_i_o_stream.rs]: Cool changes 😉

File: Gir.toml (hunk 1 above)
Comment (sdroege) [reply Gir.toml]: Like this? The list is unordered already, maybe someone wants to order it in general :)

File: src/auto/file_i_o_stream.rs (hunk 2 above)
Comment (sdroege) [reply src/auto/file_i_o_stream.rs]: Yes, boring :(

File: Gir.toml (hunk 1 above)
Comment (EPashkin) [reply Gir.toml]: `generate` array was sorted alphabetically. I missed that items added after `SimplePermission` was not in order.
Can you resort it?

Comment (sdroege) [reply Gir.toml]: Done, I hope it's in the correct order now :)

**Comment by EPashkin** [comment]
@sdroege Thanks.

**Comment by sdroege** [comment]
Then let's get it in, almost no manual code left for async functions then :)

File: src/auto/file_i_o_stream.rs (hunk 2 above)
Comment (MathieuDuponchelle) [reply src/auto/file_i_o_stream.rs]: Would be sweet if this info could be output in a separate file instead, with the option to version control it (or not) left to the repo maintainer

**Comment by sdroege** [comment]
@GuillaumeGomez anything missing from your side?

**Comment by GuillaumeGomez** [comment]
Nope, all good for me as well! Thanks!

**Pull request merged**